Project Scope Statement.md

```diff
@@ -0,0 +1,13 @@
+﻿| **PROJECT SCOPE STATEMENT** |                                                                                                                                                                                                                                                   |
+|:---------------------------:|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+|       **Project Name**      | Threewide                                                                                                                                                                                                                                         |
+|   **Project Deliverables**  | **Detailed Description**                                                                                                                                                                                                                          |
+|Minimum Viable Product (MVP 1)||
+|   Strategy learning wikis   | Threewide plateform will have a strategy learning wikis which will provide required knowledge about Tetris blocks, and other techniques. The difficulty level of these strategies will start from basic and will increase as the user progresses. |
```

Comment:
This is the other way around. MVP 2 is the wiki MVP 1 is the game with solution

Reply:
Fixed